daemon/api.go

```diff
@@ -486,6 +486,7 @@ type snapInstruction struct {
 	Channel  string       `json:"channel"`
 	LeaveOld bool         `json:"leave-old"`
 	License  *licenseData `json:"license"`
+	UserID   int          `json:"user-id"`
```

Comment:
`userID int`

Lowercase so it's not exported, and without the JSON field tag. Let's please also have a space between the JSON and non-JSON data:

`// The field below should not be unmarshalled into. Do not export them.`